guides/common/assembly_preparing-environment-for-installation.adoc

```diff
@@ -1,4 +1,4 @@
-[id="preparing-environment-for-installation"]
+[id="preparing-environment-for-satellite-installation"]
```

Comment:
This is confusing. Shouldn't the anchor reflect the same name as the file? And as long as all references within the branch use the same id that everything should work?